### ISSUE TITLE
Travis : warning during npm test

### ISSUE THREAD

**Issue by pascalgrimaud** [jhipster/generator-jhipster#2576]
Since this PR https://github.com/jhipster/generator-jhipster/pull/2565, there is a warning in our Travis build (all tests seems OK). Here some examples (click on details => **`$ $JHIPSTER_INSTALL/02-install-jhipster-stack.sh`**):
- before : https://travis-ci.org/jhipster/generator-jhipster/jobs/99825995 
- after : https://travis-ci.org/jhipster/generator-jhipster/jobs/99988659

```
(node) warning: possible EventEmitter memory leak detected. 11 keypress listeners added. Use emitter.setMaxListeners() to increase limit.
Trace
    at ReadStream.addListener (events.js:239:17)
    at ReadStream.Readable.on (_stream_readable.js:665:33)
    at new Interface (readline.js:127:11)
    at Object.exports.createInterface (readline.js:21:10)
    at Object.Interface.createInterface (/home/travis/build/jhipster/generator-jhipster/node_modules/yeoman-generator/node_modules/inquirer/node_modules/readline2/index.js:38:21)
    at PromptUI.UI (/home/travis/build/jhipster/generator-jhipster/node_modules/yeoman-generator/node_modules/inquirer/lib/ui/baseUI.js:23:40)
    at new PromptUI (/home/travis/build/jhipster/generator-jhipster/node_modules/yeoman-generator/node_modules/inquirer/lib/ui/prompt.js:26:8)
    at TestAdapter.promptModule [as prompt] (/home/travis/build/jhipster/generator-jhipster/node_modules/yeoman-generator/node_modules/inquirer/lib/inquirer.js:27:14)
    at JhipsterGenerator.Base.prompt (/home/travis/build/jhipster/generator-jhipster/node_modules/yeoman-generator/lib/base.js:227:20)
    at JhipsterGenerator.askFor (/home/travis/build/jhipster/generator-jhipster/app/index.js:437:14)
    at /home/travis/build/jhipster/generator-jhipster/node_modules/yeoman-generator/lib/base.js:429:16
    at processImmediate [as _immediateCallback] (timers.js:368:17)
```

@geraldhumphries as you did the PR, can you have a look plz? Else, I'll check this tonight or tomorrow

**Comment by geraldhumphries** [comment]
Good catch, I will look into it.

**Comment by geraldhumphries** [comment]
My PR did not fix it, so I've closed it. I will need to investigate this later.

**Comment by pascalgrimaud** [comment]
@geraldhumphries : little tip. You can link your personnal Travis account to your fork project, so you can test as long as you want, before proposing a PR. That's what I do ;)

**Comment by pascalgrimaud** [comment]
It seems this https://github.com/jhipster/generator-jhipster/pull/2592 resolves this issue.

**Comment by pascalgrimaud** [comment]
Confirmed. Warning about memory leak resolved by #2592 

Another warning appears : `require('yeoman-generator').generators.Base is deprecated. Use require('yeoman-generator').Base directly` but it's another issue

**Comment by deepu105** [comment]
Yes can you open a ticket for that, i have removed it in entity generator
and app generator need to check other subs
